vowpalwabbit/example.h

```diff
@@ -119,6 +133,7 @@ struct vw;
 struct flat_example
 {
   polylabel l;
+  float weight;  // a relative importance weight for the example, default = 1
```

Comment:
This new field needs to be added to `copy_example_metadata`

Reply:
it's already there.